src/app/extensions/quoting/pages/quote-edit/quote-edit-page.component.ts

```diff
@@ -39,4 +56,25 @@ export class QuoteEditPageComponent implements OnInit {
   addQuoteToBasket(quoteId: string) {
     this.quotingFacade.addQuoteToBasket(quoteId);
   }
+
+  private patchBreadcrumbData() {
```

Comment:
please do it via effects, see 1daca60b7e064ff2c469cc48513ed7c9ad953362 as an example